core-client/src/main/java/org/glassfish/jersey/client/internal/inject/PrimitiveToStringInserter.java

```diff
@@ -50,9 +50,8 @@
  * @author Paul Sandoz
  * @author Marek Potociar (marek.potociar at oracle.com)
  * @author Gaurav Gupta (gaurav.gupta@payara.fish)
- *
  */
-final class PrimitiveValueOfInserter implements ParameterInserter<Number, String> {
```

Comment:
Is anything dependent on this name change?

Reply:
Not as far as I can tell